src/ArticleList.jsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from "react";
+import axios from "axios";
+import ArticleItem from "./ArticleItem";
+import Footer from "./Footer";
+
+const ArticleList = ({ setNumArticles, pageNumber }) => {
+	const [isLoading, setIsLoading] = useState(true);
+	const [articles, setArticles] = useState([]);
+
+	useEffect(() => {
+		setIsLoading(true);
+		axios
+			.get("https://news-app-backend.onrender.com/api/articles", {
+				params: { p: pageNumber - 1 },
+			})
+			.then(({ data }) => {
+				setNumArticles(data.total_count);
+				setArticles(data.articles);
+				setIsLoading(false);
+			})
+			.catch((error) => console.log(error));
```

Comment:
You can remove the console log between commits - the catch is in the right place when you want to set an error in state later on